app/scripts/vendor/env-test.js

```diff
@@ -31,13 +31,7 @@
   var pwElement = document.createElement('input');
```

Comment:
All references to pwElement can be removed. The code doesn't do much right now anyways besides toggle the type, and that doesn't have much of an effect.